### DISCUSSION TITLE
我想修改托盘图标，请问该怎么办

### DISCUSSION THREAD

**Discussion by DrZhang1208** [qier222/YesPlayMusic#1315] [Q&A]
![image](https://user-images.githubusercontent.com/97793620/154398568-1fccbc52-1430-4a4a-9d4c-427e50eaeeb4.png)
这个图标一直不知道怎么修改，有没有大神教教我

**Comment by pan93412** [comment]
那個需要改掉原始檔，有一點麻煩。我晚點再來看看怎麼做。

**Reply by DrZhang1208** [reply]
好的，非常感谢大佬

**Reply by ghost** [reply]
> 那個需要改掉原始檔，有一點麻煩。我晚點再來看看怎麼做。

其实我也觉得图标不够上档次，托盘和桌面图标可以设计一个。网易云音符胶片，qq苹果音符，spotify声波，咪咕耳机。现在这个像个视频播放器。比如这类音响声音音乐等等相关，标志意境都有了，网易云还存在的原因就在这，不过失去初心回不去靠你们了。
![OIP-C](https://user-images.githubusercontent.com/72744567/161481392-f0538995-648c-479d-aa70-c03210137ab8.jpg)

**Reply by YunfangHou** [reply]
> > 那個需要改掉原始檔，有一點麻煩。我晚點再來看看怎麼做。
> 
> 其实我也觉得图标不够上档次，托盘和桌面图标可以设计一个。网易云音符胶片，qq苹果音符，spotify声波，咪咕耳机。现在这个像个视频播放器。比如这类音响声音音乐等等相关，标志意境都有了，网易云还存在的原因就在这，不过失去初心回不去靠你们了。 ![OIP-C](https://user-images.githubusercontent.com/72744567/161481392-f0538995-648c-479d-aa70-c03210137ab8.jpg)

这个图标好漂亮啊，请问是哪个平台的哪个app？

**Reply by ghost** [reply]
百度的

**Reply by tylvn** [reply]
> 那個需要改掉原始檔，有一點麻煩。我晚點再來看看怎麼做。

有没有好看点的的托盘图标，以及有没有修改的方法？
